packages/rs-drive/src/drive/grove_operations/mod.rs

```diff
@@ -235,15 +235,15 @@ impl BatchInsertTreeApplyType {
     /// - A variant of `DirectQueryType::StatelessDirectQuery` if the current type is `BatchInsertTreeApplyType::StatelessBatchInsertTree`.
     /// - `DirectQueryType::StatefulDirectQuery` if the current type is `BatchInsertTreeApplyType::StatefulBatchInsertTree`.
     /// ```
-    pub(crate) fn to_direct_query_type(&self) -> DirectQueryType {
+    pub(crate) fn to_direct_query_type(self) -> DirectQueryType {
```

Comment:
We we consume self here it should be called `into_direct_query_type`

Reply:
self implements copy; so there's no need to take as ref

Reply:
@QuantumExplorer what do you think?

Reply:
Pasta is right according to linter.